docs/geode-native-docs/function-execution/handling-function-results.html.md.erb

```diff
@@ -19,7 +19,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-Geode provides a default result collector. If you need special results handling, code a custom `ResultsCollector` implementation to replace the provided default. Use the `Execution::withCollector` method to define your custom collector.
+There is a default result collector. If you need special results handling, code a custom `ResultsCollector` implementation to replace the provided default. Use the `Execution::withCollector` method to define your custom collector.
```

Comment:
As long as you (@karensmolermiller) are satisfied that no meaning was lost, you have my +1 for this PR.